packages/database/lib/index.ts

```diff
@@ -1,13 +1,13 @@
-import { drizzle, NodePgDatabase } from 'drizzle-orm/node-postgres';
-import * as schemaDefs from './schema';
-import { Pool } from 'pg';
-import 'dotenv/config'
+import { drizzle, NodePgDatabase } from "drizzle-orm/node-postgres";
+import * as schemaDefs from "./schema";
+import { Pool } from "pg";
+import "dotenv/config";
 
-let connectionString = process.env.PRODUCTION_FUTARCHY_PG_URL
+let connectionString = process.env.PRODUCTION_FUTARCHY_PG_URL;
```

Comment:
why do we need it like this? Can't we have just one URL?

Reply:
Yea but this was before I had the different systems AND locally.... lulz

Reply:
Ah I see. okay so I can switch it back to one URL then?